core/src/main/scala/sttp/example/Main.scala

```diff
@@ -0,0 +1,14 @@
+package sttp.example
+
+import org.openapitools.client.api.StoreApi
+import org.openapitools.client.core.{ApiKeyValue, SttpSerializer}
+import sttp.client.HttpURLConnectionBackend
+
+object Main {
+  def main(args: Array[String]): Unit = {
+    implicit val serializer: SttpSerializer = new SttpSerializer()
+    implicit val apiKeyValue = ApiKeyValue("")
+    implicit val backend = HttpURLConnectionBackend()
+    println(StoreApi().getInventory().send())
```

Comment:
ah the server address is in the yaml ... but there is a way to overwrite it here, if we wanted to call another address?

Reply:
Yes, this is that default parameter which is defined inside of an `apply` method from the companion object